packages/react-instantsearch-core/src/connectors/__tests__/connectInfiniteHits.js

```diff
@@ -520,14 +509,6 @@ describe('connectInfiniteHits', () => {
       expect(state1).toEqual({ page: 6 });
     });
 
-    it('automatically converts String state to Number', () => {
```

Comment:
no need to test `refine` and this behavior doesn't need to be tested because refineNext and refinePrevious already pass a number to it.